lib/compat/wordpress-6.1/get-global-styles-and-settings.php

```diff
@@ -33,16 +45,23 @@ function wp_add_global_styles_for_blocks() {
 				array_filter(
 					$metadata['path'],
 					function ( $item ) {
-						if ( str_contains( $item, 'core/' ) ) {
+						if ( str_contains( $item, '/' ) ) {
```

Comment:
I think to check only slashes is the only way to include non-core blocks, is there a better approach?